static/panes/compiler.ts

```diff
@@ -2416,6 +2416,17 @@ export class Compiler extends MonacoPane<monaco.editor.IStandaloneCodeEditor, Co
         this.statusIcon = this.domRoot.find('.status-icon');
 
         this.monacoPlaceholder = this.domRoot.find('.monaco-placeholder');
+
+        $(this.domRoot).on('keydown', event => {
+            if ((event.ctrlKey || event.metaKey) && String.fromCharCode(event.which).toLowerCase() === 's') {
+                event.preventDefault();
+                if (this.assembly.length > 0) {
+                    const texts = this.assembly.map(asm => (asm.text ? asm.text : ''));
```

Comment:
Super minor but it should be possible to write `asm => (asm.text ? asm.text : '')` as `asm => asm.text ?? ''`